packages/dev/core/src/Loading/sceneLoader.ts

```diff
@@ -777,13 +804,13 @@ function importMesh(
     pluginExtension: Nullable<string> = null,
     name = "",
     pluginOptions: PluginOptions = {}
-): Nullable<ISceneLoaderPlugin | ISceneLoaderPluginAsync> {
+): Promise<Nullable<ISceneLoaderPlugin | ISceneLoaderPluginAsync>> {

```

Comment:
Why change this? It doesn't seem like there is any new asynchrony here, and it introduces a breaking change.

Reply:
Function is flagged with asynchronous keyword now hence the promise